CHANGELOG.md

```diff
@@ -12,13 +12,31 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Breaking changes
 
-* Commit templates no longer normalize `description` by appending final newline
-  character. Use `description.trim_end() ++ "\n"` if needed.
+### Deprecations
+
+### New features
+
+### Fixed bugs
+
+### Packaging changes
+
+
+## [0.31.0] - 2025-06-02
+
+### Release highlights
+
+Revsets now better support looking for hidden commits, as well as looking
+for revisions by prefix of commit/change IDs.
```

Comment:
nit: I find that this reads a bit oddly? Honestly, I also think it's OK to have no highlights this release, it's been a quiet summer so far. (We could either remove this section entirely or just say "There are no major highlights in this release."

Reply:
I think you're right, we shouldn't feel obligated to have something to highlight each release. I got rid of the section.